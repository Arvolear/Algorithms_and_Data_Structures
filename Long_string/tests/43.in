68595523849099195853448204
690270965418479468676441099395574