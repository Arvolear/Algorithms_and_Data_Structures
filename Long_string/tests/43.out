690271034014003317775636952843778
690270896822955619577245245947370
47349498470704034676436865678675646329681783353868115849096
10062915